pkg/daemon/rpm-ostree.go

```diff
@@ -196,6 +196,48 @@ func podmanInspect(imgURL string) (imgdata *imageInspection, err error) {
 
 }
 
+// skopeoInspect is a wrapper around the "skopeo inspect" command. Skopeo gets the proxy
+// from our environment, reads /etc/containers/registries.* so it should be mirror-aware
+// and it handles exponential backoff/retries that take the type of error into account. It's preferred
+// over podmanInspect because it doesn't have to pull the image to inspect it.
+func skopeoInspect(imgURL string) (imgdata *imageInspection, err error) {
+	// TODO(jkyros): This does not handle manifestlists. It looks like if you
+	// run with --raw, you can get the manifests and inspect them separately
+	// but we don't need that yet.
+
+	// Pull the container image if not already available
```

Comment:
Does this actually pull the image?

Reply:
It does not, I stole that block from podmanInspect and retooled it for skopeo. Just missed that comment, I'll clean it out.